index.js

```diff
@@ -1,18 +1,19 @@
-import express from 'express';
+import expresponses from 'express';
```

Comment:
`import express from 'express` was fine here though

Reply:
This feedback has been implemented now.